catalog/app/containers/Bucket/PackageCopyDialog.js

```diff
@@ -0,0 +1,399 @@
+import * as R from 'ramda'
+import { FORM_ERROR } from 'final-form'
+import * as React from 'react'
+import * as RF from 'react-final-form'
+import * as M from '@material-ui/core'
+
+import AsyncResult from 'utils/AsyncResult'
+import * as APIConnector from 'utils/APIConnector'
+import * as AWS from 'utils/AWS'
+import * as Data from 'utils/Data'
+import Delay from 'utils/Delay'
+import * as NamedRoutes from 'utils/NamedRoutes'
+import Skeleton from 'components/Skeleton'
+import StyledLink from 'utils/StyledLink'
+import tagged from 'utils/tagged'
+import * as workflows from 'utils/workflows'
+import * as validators from 'utils/validators'
+
+import * as PD from './PackageDialog'
+import * as requests from './requests'
+
+async function requestPackageCopy(
```

Comment:
async is unnecessary here, bc you return Promise anyways and dont use `await` in the body